Library/Homebrew/cask/dsl.rb

```diff
@@ -2,6 +2,7 @@
 
 require "locale"
 require "lazy_object"
+require "licensee"
```

Comment:
You should avoid requiring/using `licensee` outside of `Library/Homebrew/dev-cmd/audit.rb` or `Library/Homebrew/cask/cmd/audit.rb`.